src/v0/sources/formsort/transform.js

```diff
@@ -0,0 +1,32 @@
+const path = require("path");
+const fs = require("fs");
+const { generateUUID } = require("../../util");
+const Message = require("../message");
+
+// import mapping json using JSON.parse to preserve object key order
+const mapping = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, "./mapping.json"), "utf-8")
+);
+
+function process(event) {
+  const message = new Message(`Formsort`);
+
+  // we are setting event type as track always
+  message.setEventType("track");
+
+  // setting anonymousId
+  message.anonymousId = generateUUID();
```

Comment:
No need to generate anonymousId if userId is populated.

Reply:
done